### PR TITLE
copy(fxa-auth-server): update postAddTwoStepAuthentication email copy

### PR DESCRIPTION
**Pull request by millsoper**
## Because:

* We're updating email copy

## This commit:

* Update copy, tests, and fluent files for postAddTwoStepAuthentication email

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
Screen Shot 2022-10-04 at 12 14 38 PM

### CODE REVIEW

**Review by bcolsson — APPROVED** [review]

**Review by dschom — CHANGES_REQUESTED** [review]
One minor nitpick, there appear to be new lines missing in the plain text. Please fix this before merging. Other than this it looks good 👍 .

image.

By the way, I think you have provided a screen shot of a different template by accident.

File: packages/fxa-auth-server/lib/senders/emails/templates/postAddTwoStepAuthentication/en.ftl [hunk 1]
```diff
@@ -1,6 +1,6 @@
-postAddTwoStepAuthentication-subject = Two-step authentication enabled
-postAddTwoStepAuthentication-title = Two-step authentication enabled
-postAddTwoStepAuthentication-description-plaintext = You have successfully enabled two-step authentication on your { -product-firefox-account }. Security codes from your authentication app will now be required at each sign-in.
-postAddTwoStepAuthentication-description = You have successfully enabled two-step authentication on your { -product-firefox-account } from the following device:
+postAddTwoStepAuthentication-subject-2 = Two-step authentication turned-on
```
[the remark this answers is not in the document]
Comment (flodolo) [reply packages/fxa-auth-server/lib/senders/emails/templates/postAddTwoStepAuthentication/en.ftl:1]: This should probably be without an hyphen
https://www.merriam-webster.com/dictionary/turned-on

**Comment by millsoper** [comment]
> One minor nitpick, there appear to be new lines missing in the plain text. Please fix this before merging. Other than this it looks good 👍 .
> 
> image.
> 
> By the way, I think you have provided a screen shot of a different template by accident.

Good idea! Updated and thank you!

**Review by dschom — APPROVED** [review]
LGTM!

**Pull request merged**